src/widgets/helper/ChannelView.hpp

```diff
@@ -83,6 +84,17 @@ class ChannelView final : public BaseWidget
     const boost::optional<MessageElementFlags> &getOverrideFlags() const;
     void updateLastReadMessage();
 
+    /**
+     * Attempts to scroll to a message in this channel.
+     * @return <code>true</code> iff the message was found and highlighted.
+     */
+    bool scrollToMessage(const MessagePtr &message);
+    /**
+     * Attempts to scroll to a message id in this channel.
+     * @return <code>true</code> iff the message was found and highlighted.
+     */
+    bool scrollToMessageId(const QString &id);
```

Comment:
Just use `if` instead of `iff` here

Reply:
Done in df19120fe14c32ff7f125e48e8ceda399edd01d5.